### PR TITLE
update readme to reflect current best practices

### PR DESCRIPTION
**Pull request by jelhan**
Please find details for all changes in individual commit messages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,13 +279,14 @@ If an adapter implements specific methods you wish to call, then you can use `in
...
+import { Route } from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default Ember.Route.extend({
-  metrics: Ember.inject.service(),
+export default Route.extend({
+  metrics: service(),
   afterModel(model) {
-    const metrics = Ember.get(this, 'metrics');
-    const id = Ember.get(model, 'googleAnalyticsKey');
```
Comment (mike-north) [inline README.md]: What's the grounds for changing from const to let here?

Comment (jelhan) [reply README.md]: Had the feeling that using `const` only on the top of a file is an established pattern in ember ecosystem but could revert if you prefer.

Comment (mike-north) [reply README.md]: > Had the feeling that using `const` only on the top of a file is an established pattern in ember ecosystem but could revert if you prefer.

I'd try to leave const/let alone, for the same reason that I'd avoid changing tabs to spaces or vice versa

Comment (jelhan) [reply README.md]: Reverted the change.

In general I would recommend to follow the patterns enforced by default ember blueprints (indentation by two spaces) and used by ember guides (prefer `let` over `const`). But I don't care that much to be honest.

**Pull request merged**